src/test/java/com/client/core/TestSessionFilter.java

```diff
@@ -43,9 +42,9 @@ public void testDoFilterBlockCore() throws IOException, ServletException {
 		this.req.setServletPath("/main/files/getFiles");
 		this.req.setMethod("GET");
 		this.sessionFilter.doFilter(req, rsp, mockChain);
-		assertTrue(
-				"The SessionFilter is not setup correctly. Security is compromised. Please make sure the web.xml has the SessionFilter setup for all relevant urls.",
-				rsp.getStatus() == HttpStatus.UNAUTHORIZED.value());
+		Assertions.assertTrue(rsp.getStatus() == HttpStatus.UNAUTHORIZED.value(),
```

Comment:
I believe there is an `Assertions.assertEquals(Integer a, Integer b, String message);` function that can be used here